### ISSUE TITLE
Extrafields `sellist` - Why labels are truncated to 18 chars ?

### ISSUE THREAD

**Issue by tof06** [Dolibarr/dolibarr#14945]
Dolibarr version : 12.0.3

```php
// Extrafields.class.php
 // Several field into label (eq table:code|libelle:rowid)
  $notrans = false;
  $fields_label = explode('|', $InfoFieldList[1]);
  if (is_array($fields_label) && count($fields_label) > 1) {
       $notrans = true;
       foreach ($fields_label as $field_toshow) {
              $labeltoshow .= $obj->$field_toshow.' ';
        }
  } else {
        $labeltoshow = $obj->{$InfoFieldList[1]};
  }
  $labeltoshow = dol_trunc($labeltoshow, 45);

  if ($value == $obj->rowid) {
      if (!$notrans) {
	  foreach ($fields_label as $field_toshow) {
	      $translabel = $langs->trans($obj->$field_toshow);
	      $labeltoshow = dol_trunc($translabel, 18).' ';
	   }
        }
        $out .= '<option value="'.$obj->rowid.'" selected>'.$labeltoshow.'</option>';
   } else {
         if (!$notrans) {
            $translabel = $langs->trans($obj->{$InfoFieldList[1]});
            $labeltoshow = dol_trunc($translabel, 18);
   }
                               
```

Is there any reason for truncating labels in extrafields lists ?

Thanks a lot

**Comment by tof06** [comment]
Oh Sorry. Just seen that it has been fixed in develop (e89bfe5ac1185bcd33c8a5d6c770ddb2bc098840).
Too bad it hasn't be merged in 12.0 branch before 12.0.3 
Will it be in next release ?

**Comment by eldy** [comment]
Will be in v13
